Dockerfiles/go-app-common.Dockerfile

```diff
@@ -0,0 +1,6 @@
+ARG BASE_IMAGE=debian:bookworm-slim
+
+FROM --platform=$TARGETPLATFORM ${BASE_IMAGE}
+ARG BINARY_TO_ADD
+
+ADD ${BINARY_TO_ADD} /
```

Comment:
binaries should go in /usr/local/bin
add `CMD` or `ENTRYPOINT`

Reply:
done